docs/products/flink/howto/create-integration.rst

```diff
@@ -0,0 +1,60 @@
+Create Flink integrations
+===================================
+
+Apache Flink can create streaming data pipelines across services. 
+
+Currently Aiven for Flink supports Aiven for Apache Kafka and Aiven for PostgreSQL as source and target for Flink jobs.
+
+Create a Flink integration
+--------------------------
+
+The integration between Flink and other Aiven services can be created using :doc:`the Aiven cli</docs/tools/cli>` and the Aiven console.
+
+Create a Flink integration via Aiven CLI
+''''''''''''''''''''''''''''''''''''''''
+
+To create a Flink integration via Aiven CLI, you can use the `avn service integration-create` command.
+
+**Example:** Create an integration between an Aiven for Flink cluster named ``flink-demo`` and an Aiven for Kafka cluster named ``kafka-demo``.
+
+::
+
+  avn service integration-create    \
+    --project test                  \
+    -t flink                        \
+    -s kafka-demo                   \
+    -d flink-demo
+
+The integration id, required for the following steps can be fetched with::
```

Comment:
Which next steps? I'm confused.

Also I think "id" should be "ID", it's an acronym. But I don't think the spell checker was bothered ...

Reply:
Fixed!